units/patch_intent.go

```diff
@@ -766,6 +762,28 @@ func (j *patchIntentProcessor) buildTriggerPatchDoc(ctx context.Context, patchDo
 		}
 	}
 
+	if patchDoc.IsChild() && !intent.ShouldFinalizePatch() {
```

Comment:
why do we subscribe here and in FinalizePatch?

Reply:
Oof, yeah, this doesn't look right. I think at one point we discussed moving it to the end of finalize so that it doesn't create the subscription if the patch fails to finalize, and I accidentally left it in both places. I'll remove it from here and leave it in finalizePatch.

Reply:
Thanks for spotting this!